data/wiki/hello-world.md

```diff
@@ -302,7 +302,7 @@ To run your application, execute the binary — and observe the exciting log
 messages that our loop is generating:
 
 ```bash
-$ ./hello
+$ dist/hello
```

Comment:
Minor nitpick: while a leading `./` isn't usually necessary it makes it a bit clearer where the file is.
```suggestion
$ ./dist/hello
```